src/applications/vaos/tests/mocks/helpers.js

```diff
@@ -112,3 +113,154 @@ export function mockVACancelFetches(id, reasons) {
     { data: {} },
   );
 }
+
+export function mockParentSites(ids, data) {
+  setFetchJSONResponse(
+    global.fetch.withArgs(
+      `${environment.API_URL}/vaos/v0/facilities?${ids
+        .map(id => `facility_codes[]=${id}`)
+        .join('&')}`,
+    ),
+    { data },
+  );
+}
+
+export function mockSupportedFacilities({
+  siteId,
+  parentId,
+  typeOfCareId,
+  data,
+}) {
+  setFetchJSONResponse(
+    global.fetch.withArgs(
+      `${
+        environment.API_URL
+      }/vaos/v0/systems/${siteId}/direct_scheduling_facilities?type_of_care_id=${typeOfCareId}&parent_code=${parentId}`,
+    ),
+    { data },
+  );
+}
+
+export function mockEligibilityFetches({
```

Comment:
These fetches are all typically make together or close together, so I figured this would be an easier interface than having functions for each.